conf/sample.ini

```diff
@@ -713,6 +713,8 @@
 ;allowed_groups =
 ;role_attribute_path =
 ;role_attribute_strict = false
+; org_attribute_path =
```

Comment:
Do we need the attribute path for okta?

Reply:
Yes, because on Okta there is a possibility to create custom attributes (and claims) and I think it makes sense to add the `org_attribute_path` support to it!